src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.tt

```diff
@@ -0,0 +1,135 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ output extension=".g.cs" #>
+<#@ assembly name="System.Core" #>
+<#@ assembly name="$(SonarCSharpDllPath)" #>
+<#@ assembly name="$(SonarVBDllPath)" #>
+<#@ assembly name="$(NuGetPackageRoot)Microsoft.CodeAnalysis.Common\3.3.1\lib\netstandard2.0\Microsoft.CodeAnalysis.dll" #>
```

Comment:
Just wondering if this can be used with GeneratePathProperty too as an MSBuild property